src/post/sponsorship.rs

```diff
@@ -5,27 +5,15 @@ use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
 use near_sdk::serde::{Deserialize, Serialize};
 use near_sdk::{AccountId, Balance, Timestamp};
 use std::collections::HashSet;
-use std::str::FromStr;
 
 #[derive(BorshSerialize, BorshDeserialize, Serialize, Deserialize, Clone)]
 #[serde(crate = "near_sdk::serde")]
 pub enum SponsorshipToken {
-    Near,
+    NEAR,
     NEP141 { address: AccountId },
     USD,
 }
 
-impl FromStr for SponsorshipToken {
-    type Err = ();
-
-    fn from_str(s: &str) -> Result<Self, Self::Err> {
-        match s.to_lowercase().as_str() {
-            "near" => Ok(Self::Near),
-            _ => Ok(Self::NEP141 { address: s.to_string() }),
-        }
-    }
-}
```

Comment:
This was not used, but it could have confused readers that it could handle JSON deserialization.